integration-cli/requirements_test.go

```diff
@@ -109,6 +117,9 @@ func Network() bool {
 }
 
 func Apparmor() bool {
+	if strings.HasPrefix(testEnv.DaemonInfo.OperatingSystem, "SUSE Linux Enterprise Server ") {
```

Comment:
@thaJeztah yes. I asked @andrewhsu and it seems that we don't really support apparmor stuff on SUSE.

Reply:
In that case; should we disable it on the host?

Reply:
@thaJeztah I agree it's not ideal. Maybe we could check the existence of the apparmor profile for docker instead of just checking whether apparmor is enabled. Do you think we can do that in a followup?